### DISCUSSION TITLE
[Storage] Is image/video resizing and thumbnail support coming June?

### DISCUSSION THREAD

**Discussion by tomasroaldsnes** [supabase/supabase#6867] [General]
Saw a previous post somewhere that more Cloudinary-like support for asset resizing and especially thumbnail support for videos is coming in June 2022? Is this correct? 

Please add this 😄 Would love to have this, even if it is a paid feature.

**Comment by IsNotASword** [comment]
I need this. I'd pay for it! This is what makes me want to move out of Supabase to AWS.

**Comment by fenos** [comment]
It's coming very soon! <3

**Reply by nkeat12** [reply]
status here>
